perceval/backends/github.py

```diff
@@ -111,13 +148,42 @@ def fetch_from_cache(self):
 
         cache_items = self.cache.retrieve()
 
-        for raw_issues in cache_items:
-            issues = json.loads(raw_issues)
+        issues = []
+        # In the cache first is the item with issues and next users data
+        # Read issues and their users and their orgs before processing issues
+        for raw_item in cache_items:
```

Comment:
IMO, it makes more sense to have a `while` loop here. When you find a user item in the cache, you know the next item will be an organization item. Therefore, you can read a pair of items in sequence to build a `user`data dict. In fact, you can build each item in the same loop, instead of splitting the process into three loops.